composer.json

```
@@ -123,6 +124,7 @@
             "AsyncAws\\Rekognition\\Tests\\": "src/Service/Rekognition/tests",
             "AsyncAws\\Route53\\Tests\\": "src/Service/Route53/tests",
             "AsyncAws\\S3\\Tests\\": "src/Service/S3/tests",
+            "AsyncAws\\Scheduler\\Tests\\": "src/Service/Schedulertests",
```

This is why tests are overriden by the generator

```suggestion
            "AsyncAws\\Scheduler\\Tests\\": "src/Service/Scheduler/tests",
```